server/models/users_test.go

```diff
@@ -0,0 +1,93 @@
+package models
+
+import (
+	"context"
+	"log"
+	"testing"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/jmoiron/sqlx"
+	pb "github.com/naoyakurokawa/app-grpc-web/hello"
+	"golang.org/x/crypto/bcrypt"
+)
+
+const (
+	Name     = "テスト"
+	Score    = 123
+	Photourl = "https://test"
+	Password = "abcd12341231"
+)
+
+// ユーザ取得 正常
+func TestGetUsersSuccess(t *testing.T) {
+	db, err := sqlx.Open("mysql", "root:test@tcp(127.0.0.1:13306)/test")
+	if err != nil {
+		log.Fatalln(err)
```

Comment:
こちらも t.Error を使っていただくのが一般的かと思います！
```suggestion
		t.Error("failed to open mysql connection: %v", err)
```